packages/react/src/components/AccountSettings/types.ts

```diff
@@ -6,29 +6,45 @@ import {
   PrimitiveProps,
 } from '../../primitives/types';
 
-/**
+/*
  * These are primitive prop types that account settings component use.
  *
  * Note that `PrimitiveProps` is used to get native html types, like `onSubmit`.
  */
-type CommonPasswordFieldProps = Partial<
-  PrimitiveProps<PasswordFieldProps, 'input'>
->;
-type CommonButtonProps = Partial<PrimitiveProps<ButtonProps, 'button'>>;
-type CommonAlertProps = Partial<PrimitiveProps<AlertProps, 'div'>>;
+type PasswordFieldPrimitiveProps = PrimitiveProps<PasswordFieldProps, 'input'>;
+type ButtonPrimtiveProps = PrimitiveProps<ButtonProps, 'button'>;
```

Comment:
```suggestion
type ButtonPrimitiveProps = PrimitiveProps<ButtonProps, 'button'>;
```

Then fix below as well

Reply:
This is an interesting pattern using the `PrimitiveProps` nice

Reply:
Thanks for catching it, [37ded7e](https://github.com/aws-amplify/amplify-ui/pull/3131/commits/37ded7e14ac49da49152ba91488a57584b058d86)